cross/mutt/Makefile

```diff
@@ -1,28 +1,29 @@
 PKG_NAME = mutt
-PKG_VERS = 1.5.24
+PKG_VERS = 1.13.4
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = ftp://ftp.mutt.org/pub/$(PKG_NAME)
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
-DEPENDS = cross/ncursesw cross/openssl
+DEPENDS = cross/ncursesw cross/openssl cross/cyrus-sasl
 
 HOMEPAGE = http://www.mutt.org
 COMMENT  = Mutt is a small but very powerful text-based mail client for Unix operating systems.
 LICENSE  = GPLv2
 
 GNU_CONFIGURE = 1
-CONFIGURE_ARGS = --with-ssl=openssl --enable-pop --enable-imap --enable-smtp
+CONFIGURE_ARGS = --with-ssl=openssl --enable-pop --enable-imap --enable-smtp --with-sasl
+
+INSTALL_TARGET = mutt_install
+POST_INSTALL_TARGET = mutt_post_install
 
-INSTALL_TARGET = myInstall
-POST_INSTALL_TARGET = myPostInstall
 
 include ../../mk/spksrc.cross-cc.mk
 
-.PHONY: myInstall
-myInstall:
+.PHONY: mutt_install
+mutt_install:
 	$(RUN) $(MAKE) install DESTDIR=$(INSTALL_DIR)
 
-.PHONY: myPostInstall
-myPostInstall:
+.PHONY: mutt_post_install
+mutt_post_install:
 	cp $(WORK_DIR)/$(PKG_DIR)/doc/Muttrc $(STAGING_INSTALL_PREFIX)/etc/Muttrc
```

Comment:
I would group these two actions in a single target for simplicity.
POST_INSTALL target is a hook to process tasks after original INSTALL target. If package replace INSTALL target, it make sense to put all processing there. (and so using POST_INSTALL hook no longer makes sense)